x-pack/plugins/security_solution/public/common/hooks/flyout/use_init_flyout_url_param.ts

```diff
@@ -42,7 +42,7 @@ export const useInitFlyoutFromUrlParam = () => {
     const { initialized, isLoading, totalCount, additionalFilters } = dataTableCurrent;
     const isTableLoaded = initialized && !isLoading && totalCount > 0;
     if (urlDetails) {
-      if (!additionalFilters.showBuildingBlockAlerts) {
+      if (!additionalFilters || !additionalFilters.showBuildingBlockAlerts) {
```

Comment:
I remember facing this issue during Alert Table refactoring and created this hook : `x-pack/plugins/security_solution/public/common/hooks/use_data_table_filters.ts` for safe access to `additionalFilters`

Reply:
Yea, that makes sense. Since I'm already accessing the entire table state here, I'll leave it as is for now, but thanks for the tip!